templates/admin/usage_dashboard.html

```diff
@@ -0,0 +1,42 @@
+{% extends "/base.html" %}
+{% set active_page = "profile" %}
+{% set active_link = "admin" %}
+
+{% block content %}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css">
+<script src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script>
```

Comment:
Are we going to enhance this page with chart?

Reply:
If this is easy to do, sure! A chart with the horizontal axis being each component name and the vertical axis being the counts.

If not, we can save it for a separate enhancement.